app/static/js/app.js

```diff
@@ -3,6 +3,7 @@
 const socket = io();
 let connectedToServer = false;
 let keystrokeId = 0;
+var settings = { cursor: 3 };
```

Comment:
I'm having trouble understanding this. It seems like we're saving in case of a restart, but doesn't the page reload wipe the global JS context anyway?

Reply:
This is being used to remember the current state, a todo would be to save to localstorage so its persistent. I am currently using it to hold the state for the connect/disconnect of the sockets not page reloads.